docs/docs/installation/running-on-kubernetes.mdx

```diff
@@ -155,7 +155,7 @@ connect to those datasources in your Superset installation:
 ```yaml
 bootstrapScript: |
   #!/bin/bash
-  pip install psycopg2==2.9.6 \
+  pip install psycopg2==2.9.9 \
```

Comment:
this should probably reference the optional dependencies I referenced in another comment. Maybe should say something like `pip install -e .[postgres]`, which would make it consistent and in sync with the ranges defined in pyproject.toml